sources/europe/sk/OrtofotomozaikaSR.geojson

```diff
@@ -12,7 +12,8 @@
         "min_zoom": 0,
         "country_code": "SK",
         "type": "tms",
-        "license_url": "https://wiki.openstreetmap.org/wiki/WikiProject_Slovakia/Sources#Ortofotomozaika_SR_.28orthophoto_by_GK.C3.9A_and_NLC.29"
+        "license_url": "https://wiki.openstreetmap.org/wiki/WikiProject_Slovakia/Sources#Ortofotomozaika_SR_.28orthophoto_by_GK.C3.9A_and_NLC.29",
+        "privacy_policy_url": "https://wiki.openstreetmap.org/wiki/WikiProject_Slovakia/Sources#Ortofotomozaika_SR_.28orthophoto_by_GK.C3.9A_and_NLC.29"
```

Comment:
How is that a privacy_policy_url?

Reply:
What is expected to be in privacy policy? Something from https://www.geoportal.sk/sk/udaje/ortofotomozaika/?

Reply:
Yes, "A URL for the privacy policy of the operator"

Reply:
What if there is nothing like that? Or I couldn't find one. Could it be this? https://www.geoportal.sk/files/zbgis/objednavky/podmienky_poskytovania_a_pouzivania_udajov_ortofotomozaiky.pdf

Reply:
No. It must be about the use of personal data (keyword GDPR). Simply remove it if there is none.

Reply:
OK, I will remove it then. Thanks.

Reply:
https://github.com/osmlab/editor-layer-index/pull/968